### PR TITLE
Terraform - reviewed and updated

### PR DESCRIPTION
**Pull request by deniseddi**
Updated vpc file in terraform. 
Added Network ACL resources. Reviewed code.

### CODE REVIEW

File: terraform/rds.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  cluster_identifier      = "wp-db"
+  engine                  = "aurora-mysql"
+  engine_version          = "5.7.mysql_aurora.2.03.2"
+  availability_zones      = ["ap-southeast-2a", "ap-southeast-2b", "ap-southeast-2c"]
+  database_name           = "wpdb"
+  master_username         = "wpadmin"
+  master_password         = "iammradmin"
+  backup_retention_period = 5
+  preferred_backup_window = "07:00-09:00"
+}
```
Comment (kikobr82) [inline terraform/rds.tf:11]: this is not part of the vpc and can be removed

File: terraform/user_data.sh [hunk 2]
```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```
Comment (kikobr82) [inline terraform/user_data.sh:1]: this file is not part of the vpc and can be removed

File: terraform/wp.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      "command": ["docker", "ps -a"],
+      "portMappings": [
+          {
+              "containerPort": 8080,
+              "hostPort": 80
+          }
+      ]
+    }
+  ]
+
```
Comment (kikobr82) [inline terraform/wp.json:18]: this is not part of the vpc and can be removed

**Review by kikobr82 — CHANGES_REQUESTED** [review]
reviewed

**Comment by starkmatt** [comment]
Looks good Denise,
Let's merge these changes to our new terraform folder instead of terraform-vpc

**Pull request closed without merging**